string obs_mode
float32 scan_num
float32 lamdel
float32 betdel